src/.vuepress/components/MaterialIcon.vue

```diff
@@ -20,14 +20,27 @@ export default {
 			type: String,
 			default: "",
 		},
-		iconName: {
+		icon: {
 			type: String,
 			required: true,
 		},
 	},
 };
 </script>
 
+<style lang="scss">
+$material-design-icons-font-directory-path: "~material-design-icons-iconfont/dist/fonts/";
+@import "~material-design-icons-iconfont/src/material-design-icons";
+
+.my-face {
+	@include material-icon("face");
+}
+
+.my-face:before {
+	content: material-icons-content("face");
+}
```

Comment:
```suggestion
.material-icon-face {
	@include material-icon("face");
}

.material-icon-face:before {
	content: material-icons-content("face");
}
```
Can the name of the CSS selector be more descriptive?

Reply:
Forgotten code, it's not necessary for our uses